api/src/main/java/org/openmrs/module/fhir2/api/translators/AppointmentTranslator.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.translators;
+
+import org.hl7.fhir.r4.model.Appointment;
+
+public interface AppointmentTranslator<T> extends ToFhirTranslator<T, Appointment>, UpdatableOpenmrsTranslator<T, Appointment> {
+	
+	/**
+	 * Maps Bahmni/OpenMRS appointment to {@link org.hl7.fhir.r4.model.Appointment}
```

Comment:
I'd remove the reference to Bahmni from here. That's more of an implementation detail.